CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ To publish a new release run `scripts/release` from the project directory.
 [GH-182]: https://github.com/lunaryorn/mdcat/pull/182
 [GH-183]: https://github.com/lunaryorn/mdcat/issues/183
 [GH-184]: https://github.com/lunaryorn/mdcat/pull/184
+[GH-190]: https://github.com/lunaryorn/mdcat/pull/190
```

Comment:
Please remove this reference definition from the 0.23.3 section to the unreleased section.